### DISCUSSION TITLE
edit form with where

### DISCUSSION THREAD

**Discussion by ehary-devs** [filamentphp/filament#9454] [Help]
`<?php

namespace App\Filament\Resources\UserResource\Pages;

use App\Filament\Resources\UserResource;
use Filament\Actions;
use App\Models\User;
use Filament\Resources\Pages\EditRecord;
use Filament\Notifications\Notification;

class EditUser extends EditRecord
{
    protected static string $resource = UserResource::class;

    protected function getHeaderActions(): array
    {
        return [
            Actions\DeleteAction::make(),
        ];
    }
    protected function getRedirectUrl(): string
    {
        return $this->getResource()::getUrl('index');
    }
    protected function getSavedNotification(): ?Notification
    {
        return Notification::make()
            ->success()
            ->title('User updated')
            ->body('The user has been saved successfully.');
    }
 
    
}`

how to display edit data under certain conditions. for example where "name"

**Comment by VincvanAs** [comment]
Could you please clarify your question? It's quite difficult to grasp what you're asking or what you aim to accomplish.
